docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -0,0 +1,605 @@
+---
+title: "SOA domain"
+date: 2019-04-18T07:32:31-05:00
+weight: 3
+description: "Sample for creating a SOA Suite domain home on an existing PV or
+PVC, and the domain resource YAML file for deploying the generated SOA domain."
+---
+
+
+The sample scripts demonstrate the creation of a SOA Suite domain home on an
+existing Kubernetes persistent volume (PV) and persistent volume claim (PVC). The scripts
+also generate the domain YAML file, which can then be used to start the Kubernetes
+artifacts of the corresponding domain. Optionally, the scripts start up the domain,
+and SOA server pods and services based on the domain type (for example, soa, osb, soaosb, soaess and soaessosb).
+
+#### Prerequisites
+
+Before you begin, read this document, [Domain resource]({{< relref "/userguide/managing-domains/domain-resource/_index.md" >}}).
+
+The following prerequisites must be handled prior to running the create domain script:
+
+* Make sure that Kubernetes is set up in the environment. For details, see [Set Up Kubernetes]({{< relref "/userguide/overview/k8s-setup.md" >}}).
+* Make sure that the WebLogic operator is running. See [Manage operators]({{< relref "/userguide/managing-operators/_index.md" >}}) for operator infrastructure setup and [Install the operator]({{< relref "/userguide/managing-operators/installation/_index.md" >}}) for operator installation.
+* The operator requires SOA Suite 12.2.1.3.0 with patch 29135930 applied. For details on how to obtain or create the image, see [SOA domains]({{< relref "/userguide/managing-domains/soa-suite/_index.md#obtaining-the-soa-suite-docker-image" >}}).
+* Create a Kubernetes namespace (for example, `soans`) for the domain unless you intend to use the default namespace. Use the same namespace in all the other steps.
+For details, see [Prepare to run a domain]({{< relref "/userguide/managing-domains/prepare.md" >}}).
+
+  ```
+$ kubectl create namespace soans
+  ```
+
+* In the same Kubernetes namespace, follow these instructions to create the Kubernetes persistent volume (PV) where the domain home will be hosted,
```

Comment:
Please remove lines 32 to 97. Then add below latest content for this step:

In the same Kubernetes namespace created above, the user needs to create the Kubernetes persistent volume (PV) where the domain home will be hosted, and the Kubernetes persistent volume claim (PVC) for the domain.

To create PV/PVC run the create-pv-pvc.sh script located at kubernetes/samples/scripts/create-weblogic-domain-pv-pvc. Please follow the instructions in the page Create sample PV and PVC (Link to https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/storage/).
The users are recommended to change the values in the inputs file based on their requirement. The inputs file is located at "kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/create-pv-pvc-inputs.yaml".
You need to ensure that the path mentioned for the property "weblogicDomainStoragePath" does exists (if not, you need to create it), has full access permissions set and also make sure that the folder does not contain any files.